src/sentry/incidents/endpoints/serializers/workflow_engine_action.py

```diff
@@ -0,0 +1,52 @@
+from sentry.api.serializers import Serializer
+from sentry.incidents.endpoints.serializers.alert_rule_trigger_action import (
+    get_identifier_from_action,
+    get_input_channel_id,
+    human_desc,
+)
+from sentry.incidents.models.alert_rule import AlertRuleTriggerAction
+from sentry.notifications.models.notificationaction import ActionService
+from sentry.workflow_engine.models import ActionAlertRuleTriggerAction
+
+
+class WorkflowEngineActionSerializer(Serializer):
+    def serialize(self, obj, attrs, user, **kwargs):
+        """
+        Temporary serializer to take an Action and serialize it for the old metric alert rule endpoints
+        """
+        from sentry.incidents.serializers import ACTION_TARGET_TYPE_TO_STRING
+
+        aarta = ActionAlertRuleTriggerAction.objects.get(action=obj.id)
+        priority = obj.data.get("priority", "")
+        type_value = ActionService.get_value(obj.type)
+        result = {
+            "id": str(aarta.alert_rule_trigger_action.id),
+            "alertRuleTriggerId": str(aarta.alert_rule_trigger_action.alert_rule_trigger.id),
```

Comment:
I am _slightly_ worried that we'll not be able to access this once we drop the `AlertRuleTrigger` table but maybe we can hang onto it until we can deprecate this endpoint.

Reply:
^^ this is the code smell that i was getting when looking at where to put the code itself.

I'm also curious, do we have to have something like this already in place for dual write? my initial thought on how we'd accomplish this task is to have the api for metric alerts handle all of this. Then when we go to remove the legacy models in the future, we can just delete the portion of the legacy endpoint that creates legacy models.